__tests__/commands.test.ts

```diff
@@ -122,4 +122,112 @@ describe('RuntimeCommandExecutor Tests', () => {
             expect(success).toEqual(false)
         })
     })
+
+    describe('isMinimumScannerPluginInstalled Tests', () => {
+        const sampleResponseJson =
+            '[\n' +
+            '  {\n' +
+            '    "name": "@salesforce/sfdx-scanner",\n' +
+            '    "version": "3.21.0",\n' +
```

Comment:
Would love to see a test comparing "3.21.0" to something like "3.9.0", or another case where the semantic ordering differs from alphabetical order.